index.src.html

```diff
@@ -811,6 +811,10 @@ <h3 id="determine-requests-referrer" dfn export>
       Let <var>referrerURL</var> be the result of <a href="#strip-url">stripping
       <var>referrerSource</var> for use as a referrer.</a>
     </li>
+    <li>
+      If the result of <a lt="url serializer">serializing</a> <var>referrerURL</var> is a string
+      whose length is greater than 4096, set <var>referrerURL</var> to <var>referrerOrigin</var>.
+    </li>
```

Comment:
You need to move this one step down since referrerOrigin isn't initialized yet. Also, you can use `<a for=string>length</a>` (= https://infra.spec.whatwg.org/#string-length) these days.

Reply:
Ugh. I should have followed up on this months ago. I addressed your feedback in https://github.com/w3c/webappsec-referrer-policy/pull/122/commits/906e3b868c61056c31d6a4d76bdf8adcd101164f a while back, and rebased today in https://github.com/w3c/webappsec-referrer-policy/pull/122/commits/039ee88ab9aa130f97ce14c87a6e7145247ff6aa.

Still ok with the verbage? If so, I'll land it.